.azure-pipelines/azure-pipelines-image-template.yml

```diff
@@ -46,13 +45,14 @@ jobs:
           sudo bash -c "echo 1 > /proc/sys/vm/compact_memory"
           ENABLE_DOCKER_BASE_PULL=y make PLATFORM=$(PLATFORM_AZP) PLATFORM_ARCH=$(PLATFORM_ARCH) configure
         displayName: 'Make configure'
+      - ${{ parameters.preSteps }}
```

Comment:
It is necessary? It will break the case, clean the slave dockers and then build again.